src/common/Validators.js

```diff
@@ -2,11 +2,17 @@ const numberRegExp = new RegExp("^[0-9]+$");
 
 export const required = value => value && value.trim() !== '' ? undefined : 'Required field';
 
-export const mustBeFormat = value => value && value.trim()[0] !== '+' ? 'Must be format: +380XXXXXXXXX' : undefined;
-
 export const mustBePhoneNumber = value => {
-    const val = value.trim()
-    return val.length > 1 && !numberRegExp.test(val.slice(1)) ? 'Must be a phone number' : undefined
+    if (value) {
+        const val = value.trim()
+        if (value.trim()[0] !== '+') {
+            return 'Must be format: +380XXXXXXXXX'
+        } else if (val.length > 1 && !numberRegExp.test(val.slice(1))) {
+            return 'Must be a phone number'
+        } else {
+            return undefined
```

Comment:
please return empty string instead of undefined

Reply:
React-Final-Form require that it should be 'undefined'